src/commands/client-setinfo.json

```diff
@@ -0,0 +1,46 @@
+{
+    "SETINFO": {
+        "summary": "Set client or connection specific info",
+        "complexity": "O(1)",
+        "group": "connection",
+        "since": "7.2.0",
+        "arity": 4,
+        "container": "CLIENT",
+        "function": "clientSetinfoCommand",
+        "command_flags": [
+            "NOSCRIPT",
+            "LOADING",
+            "STALE",
+            "SENTINEL"
+        ],
+        "acl_categories": [
+            "CONNECTION"
+        ],
+        "reply_schema": {
+            "const": "OK"
+        },
+        "arguments": [
```

Comment:
Do we want to constrain this to be a "oneof" list of enums? The wording for this command will become verbose over time in the description.

Reply:
i'm not certain i understand your intent, so maybe you should stress it.
i did consider if we at all want to show it as:
1. `SETINFO ([libname <libname>] | [libver <libver>] | [libenv <libenv>])` - like we do for any token arguments in many other commands, usually when they're optional but also non-mutually-exclusive
2. `SETINFO (libname|libenv|libver) <value>` - stressing that we're setting just one field, out of 3 options
3. `SETINFO <key> <value>` - don't document the supported field names in the JSON file, only in reids.io docs.

WDYT?

Reply:
discussed with Yossi, we'll keep 1